rest/pom.xml

```diff
@@ -846,9 +847,9 @@
         <plugin>
           <artifactId>maven-checkstyle-plugin</artifactId>
           <configuration>
-            <configLocation>../checkstyle.xml</configLocation>
+            <configLocation>${basedir}/../checkstyle.xml</configLocation>
             <propertyExpansion>
-              checkstyle.suppression.file=../checkstyle-suppressions.xml
+              checkstyle.suppression.file=${basedir}/../checkstyle-suppressions.xml
```

Comment:
And here, should we use `syndesis.basedir` here?

Reply:
yep.. I had put this in before I figured out the syndesis.basedir trick.